terraform/storage/bind/outputs.tf

```diff
@@ -1,11 +1,20 @@
 output "name" { value = google_service_account.account.name }
 output "email" { value = google_service_account.account.email }
 output "unique_id" { value = google_service_account.account.unique_id }
-output "private_key_data" {
+output "PrivateKeyData" {
   sensitive = true
   value     = google_service_account_key.key.private_key
 }
-output "project_id" { value = google_service_account.account.project }
+output "ProjectId" { value = google_service_account.account.project }
+output "private_key_data" {
+  sensitive   = true
+  value       = google_service_account_key.key.private_key
+  description = "Deprecated - The private key data of the service account"
```

Comment:
Where does this description appear?

Reply:
It doesn't appear anywhere else. This was just a heads-up that these properties are going to be deleted. I will add a comment and remove the description, which I think is more appropriate.